config/env_config.go

```diff
@@ -29,9 +28,9 @@ const (
 
 	awsSessionTokenEnvVar = "AWS_SESSION_TOKEN"
 
-	awsContainerCredentialsEndpointEnvVar     = "AWS_CONTAINER_CREDENTIALS_FULL_URI"
```

Comment:
If we're bikeshedding, then I'd prefer `*Env` - e.g. `awsContainerCredentialsEndpointEnv`